docs/topics/class-based-views/index.txt

```diff
@@ -117,9 +117,10 @@ And the view::
 
         def head(self, *args, **kwargs):
             last_book = self.get_queryset().latest('publication_date')
-            response = HttpResponse()
             # RFC 1123 date format
```

Comment:
This comment should be moved above the `headers=` argument.

Reply:
I think neither place really feels right, but I think it's slightly better there.